code/modules/power/breaker_box.dm

```diff
@@ -0,0 +1,104 @@
+// AI has ability to toggle it in 5 seconds
+// Humans need 30 seconds
+// Used for advanced grid control (read: Substations)
+
+/obj/machinery/power/breakerbox
+	name = "Breaker Box"
+	icon = 'icons/obj/power.dmi'
+	icon_state = "bbox_on"
+	var/icon_state_on = "bbox_on"
+	var/icon_state_off = "bbox_off"
```

Comment:
These should both be removed and instead have the code use an updateicon proc `icon_state = "bbox_[on]"`